torchaudio/models/_wavernn.py

```diff
@@ -1,4 +1,4 @@
-from typing import Optional
+from typing import List
```

Comment:
nit: list is not used here (yet), so let's not add it

Reply:
It has been removed.